src/databricks/labs/ucx/configure.py

```diff
@@ -0,0 +1,68 @@
+import logging
+
+from databricks.sdk.service import compute
+
+logger = logging.getLogger(__name__)
+CLUSTER_ID_LENGTH = 20  # number of characters in a valid cluster_id
+
+
+class ConfigureClusterOverrides:
+    """Installation configuration operations to suplement install.WorkspaceInstaller"""
+
+    def __init__(self, ws, _choice_from_dict):
+        self.minimum_spark_version = "13.3.x"
+        self._ws = ws
+        self._choice_from_dict = _choice_from_dict
+
+    def _valid_cluster_id(self, cluster_id: str) -> bool:
+        return cluster_id is not None and CLUSTER_ID_LENGTH == len(cluster_id)
+
+    def _configure_override_clusters(self):
+        """User may override standard job clusters with interactive clusters"""
+        logger.info("Configuring cluster overrides from existing clusters")
+
+        def is_classic(c) -> bool:
+            return (
+                c.state == compute.State.RUNNING
+                and c.spark_version > self.minimum_spark_version
+                and c.data_security_mode == compute.DataSecurityMode.NONE
+            )
+
+        def is_tacl(c) -> bool:
+            return (
+                c.state == compute.State.RUNNING
+                and c.spark_version > self.minimum_spark_version
+                and c.data_security_mode == compute.DataSecurityMode.LEGACY_TABLE_ACL
+            )
+
+        def build_and_prompt(prompt, clusters):
+            choices = {"[Default, use a job cluster]": None}
+            for _ in clusters:
+                choices[_.cluster_name] = _.cluster_id
+            return self._choice_from_dict(prompt, choices=choices)
+
+        clusters = self._ws.clusters.list(can_use_client="NOTEBOOK")
+        classic_clusters = [c for c in clusters if is_classic(c)]
+        tacl_clusters = [c for c in clusters if is_tacl(c)]
```

Comment:
please rewrite lines 44-46 as a single for loop. it was causing a bug is GroupManager before

Reply:
Done